### PR TITLE
#2456 Fix: image crop issue on attachment preview screen

### LINKED ISSUE #2456: Images are cropped on attachment preview screen
When viewing image attachment, it's top part is hidden by toolbar:

Screenshot 2023-11-28 at 21 13 12

uncropped image:
Screenshot 2023-11-28 at 21 13 34

Image should be vertically centered on attachment preview screen.

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR fixed image crop issue on attachment preview screen

close #2456 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Does not need tests (refactor only, docs or internal changes)

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [x] addresses the issue it closes (if any)
- [x] code is readable and understandable
- [x] is accompanied with tests, or tests are not needed
- [x] is free of vulnerabilities
- [x] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

File: FlowCryptCommon/Extensions/UIViewExtensions.swift [hunk 1]
```diff
@@ -8,52 +8,11 @@ public extension UIView {
...
-            topConstraint,
-            bottomConstraint,
-            leadingConstraint,
-            trailingConstraint
-        ])
+        let guide = self.safeAreaLayoutGuide
+        subview.trailingAnchor.constraint(equalTo: guide.trailingAnchor, constant: -insets.right).isActive = true
+        subview.leadingAnchor.constraint(equalTo: guide.leadingAnchor, constant: insets.left).isActive = true
+        subview.topAnchor.constraint(equalTo: guide.topAnchor, constant: insets.top).isActive = true
+        subview.bottomAnchor.constraint(equalTo: guide.bottomAnchor, constant: -insets.bottom).isActive = true
```
Comment (sosnovsky) [inline FlowCryptCommon/Extensions/UIViewExtensions.swift:15]: it seems we don't need additional inset on the bottom, as it produces additional empty space on the bottom. and also bottom corners have some strange border radius:

Screenshot 2023-12-05 at 10 29 51

**Review by sosnovsky — COMMENTED** [review]
Original issue is fixed, but needs a small fix for bottom insets

File: FlowCryptCommon/Extensions/UIViewExtensions.swift (hunk 1 above)
Comment (ioanmo226) [reply FlowCryptCommon/Extensions/UIViewExtensions.swift]: Nice catch. Just fixed

**Review by sosnovsky — APPROVED** [review]
All good now, thanks!

**Pull request merged**